src/promptflow-evals/tests/evals/e2etests/test_builtin_evaluators.py

```diff
@@ -42,18 +43,21 @@ def test_individual_evaluator_service_based(self, project_scope, azure_cred):
         assert score["violence_score"] < 1.0
         assert score["violence_reason"], "violence_reason must not be None or empty."
 
-    @pytest.mark.skip(reason="Not working in ci pipeline. For local run.")
     def test_content_safety_service_unavailable(self, project_scope, azure_cred):
         eval_fn = ViolenceEvaluator(project_scope, azure_cred)
```

Comment:
Lets add some unit test for built in evaluators to help improve coverage ?

Reply:
Discussed this already. That's planned for a follow-up PR. That was deemed not needed here for 2 reasons:
- This code was effectively already not covered, so arguably an already present problem that was only just caught by the coverage tests.
- I don't know nearly enough about mocks to properly test most of the new code in unit tests.